contrib/runners/action_chain_runner/dist_utils.py

```diff
@@ -17,16 +17,53 @@
 from __future__ import absolute_import
 import os
 import re
+import sys
 
-from pip.req import parse_requirements
+from distutils.version import StrictVersion
+
+GET_PIP = 'curl https://bootstrap.pypa.io/get-pip.py | python'
```

Comment:
This line makes me nervous. What happens if this is run on a system that does not have Internet access?

Reply:
We just advise user to run that, they can also install pip using other means.

Reply:
(and this only affects custom and advanced users which install StackStorm directly from sources and / or on an unsupported systems)

It doesn't affect users who install it using the official packages and official installer script.

Reply:
Ah good, it's not a problem then.